src/serializer/builtins.ts

```diff
@@ -74,6 +75,10 @@ export interface BuiltinTypes {
     'extended_asset?'?: ExtendedAsset
     'extended_asset[]': ExtendedAsset[]
     'extended_asset[]?'?: ExtendedAsset[]
+    block_id: BlockId
```

Comment:
We should verify this to be sure but I don't think there is a block_id builtin type so this could end up causing inconsistencies when decoding with core vs nodeos

Reply:
I'm not sure if I'm looking in the right place, but I do see this:

https://github.com/AntelopeIO/leap/blob/bd75cf0e3b8a95296a47a1d073da328c2e8854a3/libraries/chain/eosio_contract_abi.cpp#L14

I can remove it if it's not a legit type - just not sure where exactly to find the builtins for nodeos.

Reply:
Yeah hard to say, I checked now in the abieos lib and it doesn't define it. Only way to know for sure is to define an ABI that depends on the "block_id_type" and ask nodeos/cleos to decode it and see if that works

Reply:
I removed the the `BlockId` from the builtins and changed all the predefined types to use it instead of the alias 👍